app/main.py

```diff
@@ -1,3 +1,61 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from typing import List
+
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def read_config_file(file_path: str) -> dict:
+    with open(file_path, "r") as file:
+        data = json.load(file)
+    return data
+
+
+def create_customers(customers_data: List[dict]) -> list:
+    customers = [
+        Customer(
+            customer_dict["name"],
+            customer_dict["product_cart"],
+            customer_dict["location"],
+            customer_dict["money"],
```

Comment:
it's preferable to use get() method instead of parenthesis everywhere for getting value by key to avoid KeyError